src/Sylius/Bundle/AddressingBundle/Twig/CountryNameExtension.php

```diff
@@ -28,12 +28,20 @@ public function getFilters(): array
         ];
     }
 
+    /**
+     * @param CountryInterface|string $country
+     * @param string|null $locale
+     *
+     * @return string
+     */
     public function translateCountryIsoCode($country, ?string $locale = null): string
     {
-        if ($country instanceof CountryInterface) {
-            return Intl::getRegionBundle()->getCountryName($country->getCode(), $locale);
+        $countryCode = $country instanceof CountryInterface ? $country->getCode() : $country;
+
+        if ($countryName = Intl::getRegionBundle()->getCountryName($countryCode, $locale)) {
+            return $countryName;
         }
 
-        return Intl::getRegionBundle()->getCountryName($country, $locale);
+        return $countryCode ?? '';
```

Comment:
I believe it can be done even simpler:

```
if ($country instanceof CountryInterface) {
    return Intl::getRegionBundle()->getCountryName($country->getCode(), $locale) ?? $country->getCode();
}

return Intl::getRegionBundle()->getCountryName($country, $locale) ?? $country;
```

Reply:
This way the `if` block can return `null` (from `$country->getCode()`).
Also, if the `$country` argument is `null`, for example when chaining the filter on a `null` value in the teplate, the returned value will be `null` again.